CHANGELOG.md

```diff
@@ -52,6 +52,8 @@
 
 - Removed `TryFrom` conversions between `Robj` and `HashMap` for consistency. `List::into_hashmap()` and `List::from_hashmap()` should be used instead.
 
+- Added support for graphics.
```

Comment:
Perhaps mention also the feature that needs to be used to get access to graphics stuff?

Reply:
Ah, good idea. Sure.